data/aws/push-hv-data-support-files.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash -e
+
+#This files pushs all support files required for a HV production release,
+
+#When running this script, ensure the docker container is mounted to the dev_fim_share folders
+
+#Note: This files is rather unpolished and as it is Git, it must not included the
+#base HV S3 folder name or AWS credentials.
+
+usage ()
+{
+    echo 'Copying key FIM production run output files to HV.'
+    echo 'Does not included the filtered HAND files from FIM.'
+    echo 'Does included key files such as catfim, fim performance and sierra test files.'
+    echo 'Note: Please look at the code as there are assumed directories and pathing.'
+    echo '      Also.. for now, please pre-run the aws configure script for creds.'    
+    echo
+    echo 'Usage : push-hv-data-support-files.sh [REQ: -n <run name> ]'
+    echo '                                      [REQ: -hv <HV S3 bucket name>'
+    echo ''
+    echo 'REQUIRED:'
+    echo '  -n/--runName       : A name of HAND output release.'
+    echo '  -hv/--HV_S3_bucket : The root HV S3 bucket name. ie) s3://{hv....}' 
+    echo ''
+    echo 'OPTIONS:'
+    echo '  -h/--help       : help file'
+    echo
+    exit
+}
+
+while [ "$1" != "" ]; do
+case $1
+in
+    -n|--runName)
+        shift
+        runName=$1
+        ;;
+    -hv|--HV_S3_bucket)
+        shift
+        HV_S3_bucket=$1
+        ;;
+    -h|--help)
+        shift
+        usage
+        ;;
+    *) ;;
+    esac
+    shift
+done
+
+# print usage if arguments empty
+if [ "$runName" = "" ]
+then
+    echo "ERROR: Missing -n  -- run time name (fim version / folder name) argument"
+    usage
+fi
+
+if [ "$HV_S3_bucket" = "" ]
+then
+    echo "ERROR: Missing -hv  -- the HV s3 bucket name (ie: s3://{hv...})"
+    usage
+fi
+
+# FOR NOW, run the aws configure script by hand outside this tool
+read -p "Did you run aws configure and load creds already (y/n): " resp
+
+case $resp in
+    y ) echo ok, we wil proceed;;
+    n ) echo exiting...;
+         exit;;
+    * ) echo invalid response;
+         exit 1;;
+
+esac
+
+echo "Loading files to $HV_S3_bucket"
+
+echo "Copying catfim files"
+
+aws_cmd_cp="aws s3 cp"
```

Comment:
The variable `aws_cmd_cp` is defined here, but only used to build the `aws_cmd` which is only used for `echo`. I'd suggest either scrapping the variable altogether and `echo` the actual command passed to `eval` , or passing the `aws_cmd` to `eval`.